sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1320,10 +1320,6 @@ class MLPRegressor(RegressorMixin, BaseMultilayerPerceptron):
     loss_curve_ : list of shape (n_iters,)
         Loss value evaluated at the end of each training step.
 
-    t_ : int
-        Mathematically equals `n_iters * X.shape[0]`, it means
-        `time_step` and it is used by optimizer's learning rate scheduler.
```

Comment:
I think `n_iters * X.shapes[0]` part is useful information. Can we update the `t_ : ` above to include this information:

```
t_ : int
    The number of training samples seen by the solver during fitting.
    Same as `n_iter_ * n_samples`. 
```